packages/zone/src/heap.js

```diff
@@ -38,13 +35,15 @@ const detachedHeapStores = Far('heapStores', {
  */
 export const makeHeapZone = (baseLabel = 'heapZone') => {
   const { makeOnce, wrapProvider } = makeOnceKit(baseLabel, detachedHeapStores);
+
+  const subZoneProvider = (label, _options) =>
```

Comment:
```suggestion
  /**
   * @param {string} label
   * @param {any} _options
   */
  const provideSubZone = (label, _options) =>
```

Reply:
Done

Reply:
Param types added. Renaming not for stated reasons.